test/integration/basic_test.go

```diff
@@ -169,12 +169,12 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`([^"]*) should equal`, m.commandReturnShouldEqualContent)
 	s.Step(`([^"]*) should be empty`, m.commandReturnShouldBeEmpty)
 
-	s.BeforeScenario(func(interface{}) {
+	s.BeforeSuite(func() {
```

Comment:
@hferentschik This is run at the start and at the end of feature, not sure with the rest for now.
I agree to your suggestion with common_steps.go, this will save us from repeating.